packages/test/src/load/worker.ts

```diff
@@ -53,7 +61,7 @@ async function main() {
   console.log('Created worker');
 
   await worker.run();
-  await otel.shutdown();
+  await otel.shutdown().catch(console.error);
```

Comment:
This should be logged by the catch call below, do we want to ignore errors here?

Reply:
Yeah. The OTel library will error out if the collector is unavailable on shutdown which seems undesirable. I could go either way on it.